README.md

```diff
@@ -1 +1,104 @@
-# image-scan-example
\ No newline at end of file
+# Image-Scan-Example
+
+#### PREREQUISITES: 
+
+All of the CI examples rely on an existing DockerHub repo that you control and have the ability to push the container images being built. This repo should exist before getting started. This repo value will be used in the `CONTAINER_REPO` variable throughout the examples. You will also need a personal access token to be used for the DockerHub password in all pipelines.
+
+## CI Pipeline Examples
+
+The pipeline examplea use the Dockerfile inside this directory for build purposes.
+
+The `<repo_name>` tag listed in the pipeline examples file will need to be updated with a repo that is accessible via the docker login credentials outlined in the below example specific instructions.
+
+The `FALCON_CLOUD_REGION` variable may also need to be changed to accommodate your Falcon Platform region.  The current setting is for 'us-2'.
+
+These pipeline examples were built to showcase the `docker build` with a subsequent image scan handled by the CrowdStrike Image Scan API.
+
+## Jenkins Pipeline
+
+* The Jenkinsfile is intended to be built as a Jenkins Pipeline Job using 'Pipeline script from SCM' and the default settings.
+
+### Credentials
+
+* Github
+
+Github credentials will need to be added to Jenkins Global Credential Manager as the ID of 'github'. This should be the username and a personal access token added with all 'repo' and child object permissions. Personal access tokens can be created at https://github.com/settings/tokens.
+
+* ImageRegistry
+
+This example uses DockerHub as the image registry. DockerHub credentials will need to be added to the Jenkins Global Credential Manager with the ID of 'dockerhub'.
+
+* Falcon API
+
+Falcon API credentials will need to be added as two credentials in the Jenkins Global Credential Manager as Kind 'secret text' with the IDs `FALCON_CLIENT_ID` and `FALCON_CLIENT_SECRET`. OAuth2 API client and keys can be created at https://falcon.crowdstrike.com/support/api-clients-and-keys.
+
+### Jenkins Instructions
+
+1. Fork the repo
+2. Modify the `<repo_name>` tag and perhaps the `FALCON_CLOUD_REGION` to suite your needs
+3. Log into your Jenkins instance
+4. Browse to 'Manage Jenkins' -> 'Manage Credentials' -> '(global)'
+5. Add the require credentials listed [above](https://github.com/mccbryan3/image-scan-example/tree/initial_examples#credentials)
```

Comment:
Let's list the required credentials here instead of linking to "above". It makes it easier to read and prevents people from having to scroll up and then re-find their place.

Reply:
I am not sure I like that layout as it would put a large gap in the steps.